specs/engage-digital_openapi3.yaml

```diff
@@ -97,6 +98,143 @@ tags:
   - name: Surveys
 
 paths:
+  '/topologies/{topologyId}/activate':
+    put:
+      description: >-
+        This method activates an existing topology from given attributes and renders it in case of success.
+        
+        
+        Authorization: only users that are techteam.

```

Comment:
Arf, actually this isn't true anymore. The GDoc is not up to date. Everyone having the "manage topology" permission can do it.

Same for the other endpoints, none are techteam only anymore.